src/content-script.ts

```diff
@@ -124,38 +165,83 @@ async function main() {
 
 				replyCategoriesDetailsMenu.appendChild(
 					createElement("ul", {
+						"aria-labelled-by": "repository-replies-label",
 						children: [
-							createElement("li", { children: [button], role: "none" }),
+							createElement("li", {
+								children: [button],
+								role: "none",
+							}),
 						],
-						role: "none",
+						className: "js-saved-reply-menu ActionListWrap",
+						role: "list",
 					}),
 				);
 			}
 
-			replyCategoriesDetailsMenu.appendChild(
-				createElement("a", {
-					children: [
-						createElement("span", {
-							children: ["Create a new repository reply..."],
-							className: "select-menu-item-text",
-						}),
-					],
-					className: "select-menu-item select-menu-action",
-					href: `https://github.com/${userOrOrganization}/${repository}/edit/${defaultBranch}/.github/replies.yml`,
-					role: "menuitem",
-					target: "_blank",
-				}),
-			);
+			// 8. Add a second button at the bottom of the modal for adding more
+			// TODO: thanks for the heads up @keithamus :)
+			// https://github.com/primer/view_components/pull/2364
+			for (const modal of Array.from(
+				document.querySelectorAll<HTMLElement>(
+					"modal-dialog#saved_replies_menu_new_comment_field-dialog",
```

Comment:
You could keep this stable with `:where(modal-dialog, dialog).js-saved-reply-container` which probably won't change.